client/src/components/navigation/items/Search.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { Search, Responsive } from 'semantic-ui-react';
+
+const Search_ = props => {
+    return (
+        <>
+            <Responsive minWidth={Responsive.onlyTablet.maxWidth}>
```

Comment:
Is the logic correct on this line?

Reply:
What do you mean? Why is min width equals to Max Width?

Reply:
That's what I mean

Reply:
Responsive.OnlyTablet.MaxWidth is the boundary where mobile dimensions start from, so Desktop styles should begin from there and Mobile ones therefore should end there

Reply:
Everything's clear now!